pkg/flow/import_test.go

```diff
@@ -0,0 +1,769 @@
+package flow_test
```

Comment:
The River code in the tests don't render very nicely on GitHub (the indentation is inconsistent); can you format them so they're easier to follow?